<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-UFMB1VQ" failures="6" tests="93" name="com.pwc.productcentral.SanityTestCases" time="1695.472" errors="0" timestamp="2023-01-20T19:04:45 EST" skipped="0">
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_67" time="31.510"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_71" time="28.469"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_73" time="22.099"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_65" time="44.705">
    <failure message="expected [Last updated 01/09/2023] but found [Last updated 01/20/2023]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Last updated 01/09/2023] but found [Last updated 01/20/2023]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at Pages.AuthoringPage.setUpdatedDateFromAssets(AuthoringPage.java:1415)
at com.pwc.productcentral.SanityTestCases.WEB_65(SanityTestCases.java:784)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- WEB_65 -->
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_25" time="22.320"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_52_1" time="0.597"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_15" time="2.019"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_16" time="1.218"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_30" time="13.037"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_55" time="21.130"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_21" time="7.255"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_29" time="30.606"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_27" time="11.445"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_10" time="1.548"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_52" time="1.137">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at Pages.BasePage.setAllImages(BasePage.java:320)
at com.pwc.productcentral.SanityTestCases.WEB_52(SanityTestCases.java:606)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- WEB_52 -->
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_63" time="49.300"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_76" time="48.949"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_34" time="20.731"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_28" time="13.457"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_81" time="8.702"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_19" time="0.388"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_2" time="0.358"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_1" time="1.735"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_47" time="0.982"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_62" time="34.275"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_60" time="21.737"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_22" time="12.031"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_6" time="1.095"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_11" time="1.330"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_52_4" time="1.253"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_58" time="21.891">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at Pages.ProductPage.setFirstTabActive(ProductPage.java:641)
at com.pwc.productcentral.SanityTestCases.WEB_58(SanityTestCases.java:718)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- WEB_58 -->
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_79" time="22.197"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_3" time="1.103"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_36" time="28.024"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_52_5" time="22.551"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_38" time="26.200"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_12" time="38.029"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_44" time="0.629"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_50" time="15.021"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_70" time="20.559"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_74" time="38.599"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_68" time="15.919">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at Pages.HomePage.setSearchResult(HomePage.java:492)
at com.pwc.productcentral.SanityTestCases.WEB_68(SanityTestCases.java:812)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- WEB_68 -->
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_26" time="6.487"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_43" time="1.078"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_48" time="11.910"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_52_3" time="0.656"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_32" time="24.197"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_78" time="11.813"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_17" time="0.430"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_86" time="22.643"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_66" time="16.955">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at Pages.AuthoringPage.setUpdatedDateVisibility(AuthoringPage.java:1469)
at com.pwc.productcentral.SanityTestCases.WEB_66(SanityTestCases.java:793)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- WEB_66 -->
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_69" time="70.525"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_51" time="23.093"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_52_2" time="1.208"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_83" time="28.704"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_39" time="1.097"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_54" time="25.267"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_64" time="29.826"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_13" time="26.968"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_49" time="1.534"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_33" time="34.101"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_40" time="7.134"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_56" time="23.658"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_46" time="0.709">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at Pages.ProductListingPage.setDocumentTileLine(ProductListingPage.java:352)
at com.pwc.productcentral.SanityTestCases.WEB_46(SanityTestCases.java:550)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- WEB_46 -->
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_45" time="7.405"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_59" time="22.156"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_9" time="15.758"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_41" time="8.097"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_31_1" time="20.965"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_37" time="23.694"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_7" time="1.177"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_18" time="1.218"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_42" time="1.609"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_5" time="5.047"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_5_2" time="9.636"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_14" time="1.922"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_31" time="1.151"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_53" time="21.470"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_20" time="1.203"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_72" time="20.120"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_77" time="27.268"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_24" time="13.269"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_75" time="39.937"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_4" time="5.811"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_61" time="45.816"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_84" time="33.563"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_23" time="14.037"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_35" time="48.094"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_57" time="27.599"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_8" time="7.616"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_80" time="29.427"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_82" time="109.765"/>
  <system-out/>
  <testcase classname="com.pwc.productcentral.SanityTestCases" name="WEB_85" time="54.509"/>
  <system-out/>
</testsuite> <!-- com.pwc.productcentral.SanityTestCases -->
